app/representers/api/v1/unclaimed_user_representer.rb

```diff
@@ -0,0 +1,13 @@
+module Api::V1
+  class UnclaimedUserRepresenter < Roar::Decorator
```

Comment:
@Dantemss @jpslav 
I've just got through testing the use of `ApplicationUserRepresenter` vs the custom `UnclaimedUserRepresenter`.   If I use `ApplicationUserRepresenter`, the JSON that's returned is:
`{"id":4,"user":{"id":59,"username":"user379650"},"unread_updates":1}`

The original spec [issue #89](https://github.com/openstax/accounts/issues/89) for this said all that should be returned was the openstax uid.  Since that response gives out the randomly generated username, I'm not sure if we should use it.  Please advise :)